contents/blog/ab-testing-mistakes.md

```diff
@@ -0,0 +1,175 @@
+---
+date: 2023-04-27
+title: "8 A/B testing mistakes and how to avoid them"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ['lior-neu-ner']
+featuredImage: ../images/blog/experiments.jpeg
```

Comment:
```suggestion
featuredImage: ../images/blog/experiment-hog.jpg
```
The current photo is an old style of hedgehog we don't use anymore, this is a new one.